packages/compass-crud/src/components/crud-toolbar.tsx

```diff
@@ -109,6 +116,8 @@ const ERROR_CODE_OPERATION_TIMED_OUT = 50;
 const INCREASE_MAX_TIME_MS_HINT =
   'Operation exceeded time limit. Please try increasing the maxTimeMS for the query in the expanded filter options.';
 
+const COUNT_UNAVAILABLE_TOOLTIP = `The count is not available for this query. This can happen when the count operation fails or exceeds the maxTimeMS of ${COUNT_MAX_TIME_MS_CAP}.`;
```

Comment:
Though we cap maxTimeMS, but if a user has set it to lower value in preferences, should we show that instead? When running count, we call `capMaxTimeMSAtPreferenceLimit` to get actual value we use.